app/models/concerns/taxonomix.rb

```diff
@@ -219,7 +219,7 @@ def taxonomy_foreign_key_conditions
   end
 
   def used_taxonomy_ids(type)
-    return [] if new_record?
+    return [] if new_record? || !self.respond_to?(:hosts)
```

Comment:
This allows the loc_orgs tab partial to work for models that are not directly associated with hosts (such as this one, or filter, or roles which I plan to move to use the partial after this is merged)